B827EBFFFE15856A.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gateway_conf" : {
+         "gateway_ID" : "B827EBFFFE15856A",
+         "servers" : [
+              {
+                   "server_address" : "eu.thethings.network",
```

Comment:
This is not the correct address. As per [this doc](https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses) the address should be `router.eu.thethings.network`.